build/Pod.yaml

```diff
@@ -0,0 +1,18 @@
+# Example Pod for running llm-load-test on k8s/OpenShift.
+# Typical use is to oc exec/rsh into the Pod to manually run tests.
+apiVersion: v1
+kind: Pod
+metadata:
+ name: llm-load-test
+spec:
+  containers:
+  - name: llm-load-test
+    image: quay.io/dagray/llm-load-test:latest
```

Comment:
This quay repo seems out of date; can you set up an automatic build trigger?

Reply:
We should definitely look into this, and move it out of my personal quay repository

Reply:
For now, I removed the reference to this specific image. We should set up an automatic build trigger.